### DISCUSSION TITLE
Connecting to Australian Public School Internet

### DISCUSSION THREAD

**Discussion by giscus** [DeclanChidlow/vale.rocks#10] [Comments Archive]
# Connecting to Australian Public School Internet

https://vale.rocks/posts/connecting-to-australian-public-school-internet

**Comment by FanMclaine** [comment]
Imagine having such strict public school internet :trollface:

**Reply by DeclanChidlow** [reply]
It really is a pain. I can't say I'm a fan of the system, but it shouldn't bother me for much longer, for I've only got 4 days left of school!
